docs/docs/glossary/headless-cms.md

```diff
@@ -0,0 +1,22 @@
+---
+
+title: Headless CMS
+disableTableOfContents: true
+
+Learn what a Headless CMS is, how it differs from traditional content management systems, and why you might choose a headless CMS for your project.
+
+## What is a Headless CMS?
+
+A content management system, or CMS, is a software application that lets you edit and publish content for your website. Most content management systems store content in a database, and use a series of HTML-based template files to manage how that content gets presented.
```

Comment:
Begin with a single sentence that explains "headless CMS". Example: "A headless CMS is ..." 

For this audience, assume they are familiar with CMS already.